### PR TITLE
[PPP-4447] - Upgraded jackrabbit 2.16.5 to resolve issues with Tika v…

### PR DESCRIPTION
**Pull request by wseyler**
…ulnerabilities.  Also upgraded separate uses of tika.

### CODE REVIEW

**Comment by buildguy** [comment]
### :gear: Build finished in 33m 9s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit -P !assemblies
```
:ok_hand: All tests passed!
 ---

|Total|Failed|Skipped|
|---|---|---|
|2532|0|1|

|[Details](http://wingman2.pentaho.net/job/projects/job/platform/job/PR-4569/1/testReport)|
|----|


:information_source:  _This is an automatic message_

File: assemblies/pentaho-war/pom.xml [hunk 1]
```diff
@@ -22,10 +22,10 @@
     <hibernate-ehcache.version>3.6.9.Final</hibernate-ehcache.version>
     <commons-io.version>2.2</commons-io.version>
     <maven-resources-plugin.version>3.0.1</maven-resources-plugin.version>
-    <tika-core.version>1.19.1</tika-core.version>
+    <tika-core.version>1.22</tika-core.version>
     <xmlgraphics-commons.version>2.2</xmlgraphics-commons.version>
     <c3p0.version>0.9.1.2</c3p0.version>
-    <jackrabbit.version>2.14.2</jackrabbit.version>
+    <jackrabbit.version>2.16.5</jackrabbit.version>
```
Comment (pentaho-lmartins) [inline assemblies/pentaho-war/pom.xml]: Please move the version property to the parent pom repository and use dependency management.

File: assemblies/pentaho-war/pom.xml [hunk 2]
```diff
@@ -22,10 +22,10 @@
     <hibernate-ehcache.version>3.6.9.Final</hibernate-ehcache.version>
     <commons-io.version>2.2</commons-io.version>
     <maven-resources-plugin.version>3.0.1</maven-resources-plugin.version>
-    <tika-core.version>1.19.1</tika-core.version>
+    <tika-core.version>1.22</tika-core.version>
```
Comment (pentaho-lmartins) [inline assemblies/pentaho-war/pom.xml]: Are we using `tika` in our code? It is a dependency of `jackrabbit` so we should let it be included as a transitive of `jackrabbit`.

File: pom.xml [hunk 3]
```diff
@@ -210,7 +210,7 @@
     <bcmail.version>138</bcmail.version>
     <barbecue.version>1.5-beta1</barbecue.version>
     <jaxws-spring.version>1.8</jaxws-spring.version>
-    <jackrabbit.version>2.14.2</jackrabbit.version>
+    <jackrabbit.version>2.16.5</jackrabbit.version>
```
Comment (pentaho-lmartins) [inline pom.xml]: Same here.

File: extensions/pom.xml [hunk 4]
```diff
@@ -20,7 +20,7 @@
     <jstl.version>1.0.5</jstl.version>
     <license.header.definition.file>${basedir}/../license/styles/javadoc_style_license_header.xml</license.header.definition.file>
     <hamcrest-library.version>1.3</hamcrest-library.version>
-    <tika-parser.version>1.12</tika-parser.version>
+    <tika-parser.version>1.22</tika-parser.version>
```
Comment (pentaho-lmartins) [inline extensions/pom.xml]: Same here. There are several declared `tika` artifacts that are not used in our code.
Please refactor these.

**Comment by buildguy** [comment]
### :gear: Build finished in 33s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit
```
:exclamation: No tests found!


:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :gear: Build finished in 38s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit
```
:exclamation: No tests found!


:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :gear: Build finished in 28s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit
```
:exclamation: No tests found!


:information_source:  _This is an automatic message_

**Comment by wseyler** [comment]
@pentaho-lmartins Change made as requested.  This isn't building because it relies on changes in the maven-parent-poms project.

**Comment by buildguy** [comment]
### :gear: Build finished in 30s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit
```
:exclamation: No tests found!


:information_source:  _This is an automatic message_

File: assemblies/pentaho-war/pom.xml [hunk 5]
```diff
@@ -923,9 +921,9 @@
       <version>${jackrabbit.version}</version>
     </dependency>
     <dependency>
-      <groupId>org.apache.tika</groupId>
-      <artifactId>tika-core</artifactId>
-      <version>${tika-core.version}</version>
+      <groupId>org.apache.jackrabbit</groupId>
+      <artifactId>jackrabbit-core</artifactId>
+      <version>${jackrabbit.version}</version>
```
[the remark this answers is not in the document]
Comment (wseyler) [reply assemblies/pentaho-war/pom.xml]: Done.

**Comment by buildguy** [comment]
### :x: Build failed in 41s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :x: Build failed in 3m 45s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Comment by pentaho-lmartins** [comment]
Closing in favor of #4580

**Pull request closed without merging**